From fe96897b9dbfee3019846be60da4cfc4062b990e Mon Sep 17 00:00:00 2001
From: Sam Nazarko <email@samnazarko.co.uk>
Date: Sun, 23 Aug 2015 15:53:43 +0100
Subject: [PATCH] Add enumerated type OSMCSettingsState

Signed-off-by: Sam Nazarko <email@samnazarko.co.uk>
---
 xbmc/Application.cpp |  1 +
 xbmc/Application.h   | 11 +++++++++++
 2 files changed, 12 insertions(+)

diff --git a/xbmc/Application.cpp b/xbmc/Application.cpp
index a2475c6..cd283ca 100644
--- a/xbmc/Application.cpp
+++ b/xbmc/Application.cpp
@@ -268,6 +268,7 @@ CApplication::CApplication(void)
   , m_progressTrackingItem(new CFileItem)
   , m_musicInfoScanner(new CMusicInfoScanner)
   , m_fallbackLanguageLoaded(false)
+  , m_eOSMCWalkthroughState(OSMC_WALKTHROUGH_NOTRUNNING)
 {
   m_network = NULL;
   TiXmlBase::SetCondenseWhiteSpace(false);
diff --git a/xbmc/Application.h b/xbmc/Application.h
index c9ba273..556a5ff 100644
--- a/xbmc/Application.h
+++ b/xbmc/Application.h
@@ -156,6 +156,17 @@ public:
   void StopPVRManager();
   bool IsCurrentThread() const;
   void Stop(int exitCode);
+  
+  // osmc signals
+  
+  typedef enum
+  {
+	OSMC_WALKTHROUGH_NOTRUNNING = 0,
+	OSMC_WALKTHROUGH_ISRUNNING = 1,
+	OSMC_WALKTHROUGH_ISDONE = 2,
+  } OSMCWalkthroughState;
+  OSMCWalkthroughState m_eOSMCWalkthroughState;
+  void SetOSMCWalkthroughState(OSMCWalkthroughState state) { m_eOSMCWalkthroughState = state; }
    /*!
   \brief Saves settings and uptime before Kodi is stopped

-- 
2.1.4

