From cab46361e96f0e47b321732c032b02d14acfa040 Mon Sep 17 00:00:00 2001
From: Sam Nazarko <email@samnazarko.co.uk>
Date: Sat, 16 Jan 2016 17:14:16 +0000
Subject: [PATCH] Ensure thumb is disabled for hevc optimisations

Signed-off-by: Sam Nazarko <email@samnazarko.co.uk>
---
 tools/depends/target/ffmpeg/autobuild.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/depends/target/ffmpeg/autobuild.sh b/tools/depends/target/ffmpeg/autobuild.sh
index ef36bdc..3b0ce5c 100755
--- a/tools/depends/target/ffmpeg/autobuild.sh
+++ b/tools/depends/target/ffmpeg/autobuild.sh
@@ -175,6 +175,7 @@ CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" \
	--enable-zlib \
 	--disable-mipsdsp \
 	--disable-mipsdspr2 \
+        --disable-thumb \
         ${FLAGS}
 
 make -j ${BUILDTHREADS} 
-- 
2.7.4
