--- tools/depends/target/ffmpeg/autobuild.sh	2015-12-29 00:14:14.101824225 +0000
+++ /tmp/autobuild.sh	2015-12-29 00:16:14.874306577 +0000
@@ -128,9 +128,9 @@
 patch -p1 < ../ffmpeg_Speed_up_wtv_index_creation.patch
 patch -p1 < ../0001-mpeg4video-Signal-unsupported-GMC-with-more-than-one.patch
 patch -p1 < ../0001-Discard-data-before-VO-VOL-in-mpeg-4-over-mpegts.patch
-patch -p1 < ../hevcdsp_ARM_NEON_optimized_epel_functions.patch
+#patch -p1 < ../hevcdsp_ARM_NEON_optimized_epel_functions.patch
 patch -p1 < ../added_ARM_NEON_optimized_SAO_patches.patch
-patch -p1 < ../pfcd_hevc_optimisations.patch
+#patch -p1 < ../pfcd_hevc_optimisations.patch
 patch -p1 < ../add_h264_MVC_support.patch
 
 CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" \
