From 124e04938022a4ab44a7a46a622a550533a37b6c Mon Sep 17 00:00:00 2001
From: Greg McCarthy <greg@gjmccarthy.co.uk>
Date: Mon, 23 Jan 2017 14:07:24 +0000
Subject: [PATCH] Patch game addon path

---
 xbmc/addons/Addon.cpp | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/xbmc/addons/Addon.cpp b/xbmc/addons/Addon.cpp
index 3b94e5a..5d0cab3 100644
--- a/xbmc/addons/Addon.cpp
+++ b/xbmc/addons/Addon.cpp
@@ -332,6 +332,17 @@ std::string CAddon::LibPath() const
 {
   if (m_props.libname.empty())
     return "";
+
+  std::string strLibPath = URIUtils::AddFileToFolder(m_props.path, m_props.libname);
+
+  // Check if add-on library has been installed to the binaries path instead
+  std::string strSharePath = CSpecialProtocol::TranslatePath("special://xbmc/");
+  const bool bIsInSharePath = StringUtils::StartsWith(strLibPath, strSharePath);
+  if (bIsInSharePath && !CFile::Exists(strLibPath))
+  {
+    std::string strBinPath = CSpecialProtocol::TranslatePath("special://xbmcbin/");
+    strLibPath.replace(0, strSharePath.length(), strBinPath);
+  }
   return URIUtils::AddFileToFolder(m_props.path, m_props.libname);
 }
 
-- 
2.1.4

