From c90b24ec1156a935b155bbea85dcce0cd12ebaa8 Mon Sep 17 00:00:00 2001
From: Sam Nazarko <email@samnazarko.co.uk>
Date: Wed, 3 Jun 2015 14:12:58 +0100
Subject: [PATCH] Create configuration option for Vero

Signed-off-by: Sam Nazarko <email@samnazarko.co.uk>
---
 configure.ac | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/configure.ac b/configure.ac
index daec619..2a91044 100644
--- a/configure.ac
+++ b/configure.ac
@@ -677,6 +677,17 @@ case $use_platform in
      use_neon=yes
      use_cpu=cortex-a7
      ;;
+   vero)
+     use_static_ffmpeg=yes
+     use_joystick=no
+     use_sdl=no
+     use_hardcoded_tables="yes"
+     CFLAGS="$CFLAGS -pipe -O3 -funroll-loops -march=armv7-a -mtune=cortex-a9 -mcpu=cortex-a9 -mfloat-abi=hard -mfpu=neon -mabi=a$
+     CXXFLAGS="$CXXFLAGS -pipe -O3 -funroll-loops -march=armv7-a -mtune=cortex-a9 -mcpu=cortex-a9 -mfloat-abi=hard -mfpu=neon -ma$
+     FFMPEG_EXTRACFLAGS="$FFMPEG_EXTRACFLAGS -pipe -O3 -funroll-loops -march=armv7-a -mtune=cortex-a9 -mcpu=cortex-a9 -mfloat-abi$
+     use_cpu=cortex-a9
+     use_neon=yes
+     ;;
 esac
 
 if test "$target_platform" = "target_raspberry_pi" ; then
-- 
2.7.4

